plugins/common/wayfire/plugins/common/view-change-viewport-signal.hpp

```diff
@@ -8,4 +8,11 @@
 struct view_change_viewport_signal : public _view_signal
 {
     wf::point_t from, to;
+
+    /**
+     * from and to maybe null
+     * if old_viewport_invalid is false
+     * as it should trigger a full re-read
```

Comment:
```suggestion
     * Indicates whether the old viewport is known.
     * If false, then the `from` field should be ignored.
```

Reply:
I want to change the wording because whether full re-read should be triggered or not is something the plugin connecting to this signal receives.